packages/dds/tree/api-markdown-documenter.js

```diff
@@ -0,0 +1,243 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const {
+    emitMarkdown,
+    getLinkUrlForApiItem,
+    getUnscopedPackageName,
+    loadModel,
+    markdownDocumenterConfigurationWithDefaults,
+    MarkdownEmitter,
+    renderDocuments,
+    DefaultPolicies,
+} = require("@fluid-tools/api-markdown-documenter");
+const { StringBuilder } = require("@microsoft/tsdoc");
+const { ApiItemKind } = require("@microsoft/api-extractor-model");
+const chalk = require("chalk");
+const cpy = require("cpy");
+const fs = require("fs-extra");
+const path = require("path");
+const os = require("os");
+const HugoMarkdownEmitter = require("./markdown-emitter");
+
+const apiReportsDirectoryPath = path.resolve(__dirname, "_api-extractor-temp");
+const apiDocsDirectoryPath = path.resolve(__dirname, "docs", "api");
+const originalPath = path.resolve(process.argv[2]);
+
+const generatedContentNotice =
+    "[//]: # (Do not edit this file. It is automatically generated by @fluidtools/api-markdown-documenter.)";
+
+/**
+ * Creates Hugo front-matter for the given API item.
+ * This will be appended to the top of the generated API documents.
+ *
+ * @param {ApiItem} apiItem - The root API item of the document being rendered.
+ * @param {MarkdownDocumenterConfiguration} config - See
+ * {@link @fluid-tools/api-markdown-documenter#MarkdownDocumenterConfiguration}.
+ *
+ * @returns The JSON-formatted Hugo front-matter as a `string`.
+ */
+function frontMatterFromApiItem(apiItem, config) {
```

Comment:
This front-matter generation step is also Hugo-specific. Probably not needed if these docs aren't being published to fluidframework.com.